MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,276 @@
+---
+layout: post
+title: Validation in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the validation feature in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Data validation in .NET MAUI DataForm control
+
+The data form validates the data and user input in order to update the correct value in the underlying data object. In case of invalid data, the error message is shown at the bottom of the editor.
+
+## Built in validations
+
+The supported built in validations are as follows:
+
+* Data annotation
+
+#### Data annotations
+
+You can validate the data using data annotation attributes.
+
+The String type property can be validated using [Required](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.requiredattribute?view=netframework-4.8) and [StringLength](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.stringlengthattribute?view=netframework-4.8) attributes.
+
+{% tabs %}
+{% highlight C# %}
+
+    [Required(AllowEmptyStrings = false, ErrorMessage = "Name should not be empty")]
+    [StringLength(15, ErrorMessage = "Name should not exceed 15 characters")]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
```

Comment:
Refer to this [link] to know more about data annotations in `SfDataForm`.

Reply:
Add the above line before the DateRange attribute.

We can add link once annotaton topic merged

Reply:
Content added